### PR TITLE
Updated workbench and tests

### PR DESCRIPTION
**Pull request by mckenzietsean**
### Summary 
Removed the description field in exchange for a date modified field

### Details
- Removed description field
- Added date modified field
- Added new tests
### How to test?
1. Log in and go to the Workbench page
2. Create an experience
3. Save the experience
4. Return to Workbench page and check if the Date Modified field is correct
5. Edit that experience and save again
6. Return to Workbench page and check if the Date Modified field has been updated and is still correct
7. Also run some tests
### Checks 
- [ ] Tested Changes
- [ ] Stakeholder Approval

### CODE REVIEW

**Review by shushengfuqin — COMMENTED** [review]
code lgtm testing. going to look at the functionalities

**Comment by shushengfuqin** [comment]
Screen Shot 2022-03-02 at 10 49 15 AM
after using an account which have saved experiences. I see that the position of the buttons are different based on the screen size. The one i'm testing on is a 13.3-inch (2560 × 1600) macpro. I'm fine with this but is this what we want?

File: src/components/Workbench.tsx [hunk 1]
```diff
@@ -281,7 +265,7 @@ const LoadExperience = () => {
                 <DeleteIcon
                   style={{ fontSize: "36px" }}
                   onClick={() => {
-                    handleDeleteDialogOpen(index, expName);       
+                    handleDeleteDialogOpen(index, exp[0]);       
                   }}
                 />
                 <MoreHorizIcon style={{ fontSize: "36px" }} />
```
Comment (tashfia98) [inline src/components/Workbench.tsx:271]: The extra buttons can be commented out or removed for now

File: src/components/Workbench.tsx [hunk 2]
```diff
@@ -251,26 +237,24 @@ const LoadExperience = () => {
         <thead>
           <ExperienceHeader>
             <ExperienceName>EXPERIENCE NAME</ExperienceName>
-            <ExperienceDesc>DESCRIPTION</ExperienceDesc>
-            <ExperienceMisc>
-              <MoreHorizIcon style={{ fontSize: "36px" }} />
-            </ExperienceMisc>
+            <ExperienceDesc>LAST MODIFIED</ExperienceDesc>
+            <ExperienceMisc><MoreHorizIcon style={{fontSize: '36px'}}/></ExperienceMisc>
```
Comment (tashfia98) [inline src/components/Workbench.tsx:241]: The extra buttons can be commented out or removed for now

File: src/components/Workbench.tsx [hunk 3]
```diff
@@ -251,26 +237,24 @@ const LoadExperience = () => {
...
-          {expList.map((expName, index) => (
+          {expList.map((exp, index) => (
             // eslint-disable-next-line react/jsx-key
             <ExperienceRow>
-              <td>{expName}</td>
-              <td>DESCRIPTION GOES HERE</td>
+              <td>{exp[0]}</td>
+              <td>{exp[1]}</td>
               <ExperienceButtons>
                 <PlayArrowOutlinedIcon style={{ fontSize: "40px" }} />
```
Comment (tashfia98) [inline src/components/Workbench.tsx:251]: The extra buttons can be commented out or removed for now

**Review by tashfia98 — COMMENTED** [review]
Looks good, The extra buttons can be commented out or remove.

File: src/components/Workbench.tsx (hunk 1 above)
Comment (mckenzietsean) [reply src/components/Workbench.tsx:271]: Will do, however the delete button is fully functional and will remain.

**Review by shushengfuqin — APPROVED** [review]
function looks good. Sometest aren't passing.

**Pull request merged**